tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@tsconfig/node-lts/tsconfig.json",
   "compilerOptions": {
+    "module": "node18",
```

Comment:
`@tsconfig/node-lts` sets `node16` here, but we need at least `node18` in order to use [import attributes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import/with) in the `tests` package.